### PR TITLE
chore: assets and utilities clean up

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Clean up files in `packages/manager/src/assets/`  and `packages/manager/src/utilities/`
- Deleted util functions that were not used and that I don't think were worth keeping
- Removed some old unused assets
- Removed some default exports

## Preview 📷

- There should be no visual changes 🚨

## How to test 🧪

- Verify automated testing passes
- Do a quick once-over of the entire App and check for crashes and regressions

### CODE REVIEW

File: packages/manager/src/utilities/escapeStringForCLI.ts [hunk 1]
```diff
@@ -1,5 +1,3 @@
-const escapeStringForCLI = (value: string) => {
+export const escapeStringForCLI = (value: string) => {
   return value.replace(/(["'$`\\])/g, '\\$1');
 };
```
Comment (jaalah-akamai) [inline packages/manager/src/utilities/escapeStringForCLI.ts:3]: Just a potential for some more cleanup, possibly in a separate PR: This util essentially does the same as https://github.com/linode/manager/blob/develop/packages/manager/src/utilities/generate-cli.ts#L2-L10. However the one in `generate-cli` is more comprehensive in what it handles. Do we want to move forward with just one solution?

**Review by jaalah-akamai — APPROVED** [review]
Really good cleanup 🧼

**Review by mjac0bs — APPROVED** [review]
Those were some colorful distro `.png`s!

Verified that the deleted files are not used and didn't see any unexpected errors when clicking through Cloud to test on Friday; just tested Linode region functionality and that still looks good too.

File: packages/manager/src/utilities/escapeStringForCLI.ts (hunk 1 above)
Comment (bnussman-akamai) [reply packages/manager/src/utilities/escapeStringForCLI.ts:3]: Made myself a note to revisit this

**Pull request merged**